### PR TITLE
Add templates events to insert custom panel in posting editor

### PR DESCRIPTION
**Pull request by Alf007**
Event added in template posting_editor.html to include link to custom panel
Event added in template posting_layout.html to include panel template body

PHPBB3-13843

### CODE REVIEW

**Comment by Alf007** [comment]
Travis issues seems to be about: LogicException: File 'styles/prosilver/template/posting_layout.html' should not contain events, but contains: 'posting_layout_include_panel_body'..
Is this LogicException expressed somewhere ?
Event are limited to only a defined list of style templates?

**Comment by nickvergessen** [comment]
You are missing the doc entry in phpBB/docs/events.md

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -868,6 +868,13 @@ overall_header_stylesheets_after
 * Purpose: Add asset calls after stylesheets within the `</head>` tag.
 Note that INCLUDECSS will not work with this event.
 
+posting_editor_add_panel_tab
+===
+* Locations:
+    + styles/prosilver/template/posting_editor.html
+* Since: 3.1.5
```
Comment (nickvergessen) [inline phpBB/docs/events.md]: `* Since: 3.1.5-RC1`

File: phpBB/docs/events.md [hunk 2]
```diff
@@ -932,6 +939,13 @@ posting_editor_subject_before
 * Since: 3.1.0-a2
 * Purpose: Add field (e.g. textbox) to the posting screen before the subject
 
+posting_layout_include_panel_body
+===
+* Locations:
+    + styles/prosilver/template/posting_layout.html
+* Since: 3.1.5
```
Comment (nickvergessen) [inline phpBB/docs/events.md]: `* Since: 3.1.5-RC1`

**Comment by marc1706** [comment]
Replaced by #3787

**Pull request closed without merging**